midori_cart_drive_node:
  ros__parameters:
    gear_rate: 0.2
    num_of_teeth1: 47.0
    num_of_teeth2: 72.0
    tire_diameter: 208.0
    tred_width: 537.0
